pkg/karmadactl/addons/enable.go

```diff
@@ -82,5 +83,6 @@ func NewCmdAddonsEnable(parentCommand string) *cobra.Command {
 	flags.Int32Var(&opts.KarmadaEstimatorReplicas, "karmada-estimator-replicas", 1, "Karmada scheduler estimator replica set")
 	flags.StringVar(&opts.MemberKubeConfig, "member-kubeconfig", "", "Member cluster's kubeconfig which to deploy scheduler estimator")
 	flags.StringVar(&opts.MemberContext, "member-context", "", "Member cluster's context which to deploy scheduler estimator")
+	flags.StringVar(&opts.HostClusterDomain, "host-cluster-domain", kubernetes.DefaultHostClusterDomain, "The cluster domain of karmada host cluster, default to 'cluster.local' (e.g. --host-cluster-domain=host.karmada)")
```

Comment:
```suggestion
	flags.StringVar(&opts.HostClusterDomain, "host-cluster-domain", kubernetes.DefaultHostClusterDomain, "The cluster domain of karmada host cluster. (e.g. --host-cluster-domain=host.karmada)")
```

There has been a default `--host-cluster-domain='cluster.local'`, we don't need to prompt it again.
```
    --host-cluster-domain='cluster.local':
        The cluster domain of karmada host cluster, default to 'cluster.local' (e.g.
        --host-cluster-domain=host.karmada)
```